capreolus/searcher/__init__.py

```diff
@@ -267,7 +267,7 @@ def config():
         fields = "title"
 
     def query_from_file(self, topicsfn, output_path):
-        qrel_fn = "/home/xinyu1zhang/cikm/capreolus-covid/capreolus/data/covid/round=3_udelqexpand=False_excludeknown=True/ignore.qrel.txt"
+        qrel_fn = "/home/xinyu1zhang/cikm/capreolus-covid/capreolus/data/covid/round=2_udelqexpand=False_excludeknown=True/ignore.qrel.txt"
```

Comment:
any suggestions where to pass this from? add a 'doc_to_exclude' in `query_from_file ` parameter would make it clumsy for all the other searchers since they can't really do anything with it; (or we can refactor all the searcher to make all of them support doc filtering)

or another way, does it make sense to let searcher depends on benchmark? since sometimes the `topicreader` might not be `trec` but depends on the type of topic_fn in benchmark. say there we could also use `-topicreader Covid`.